### DISCUSSION TITLE
Efficiency calendar on resource for long operations

### DISCUSSION THREAD

**Discussion by evgenii-mw9** [frePPLe/frepple#685] [Q&A]
Hi, 
I have a resource that can make only 1 type of item at one time, and sales orders that can have a lot of these items in one order. For example: 1000 Items A. 
Producing this quantity can take several days, yet my two shifts have different efficiency: day shift (12 hours) 100% and night shift (12 hours) 80% on same resource.
I am trying to use efficiency calendar on the resource to get accurate manufacturing order durations, but it seems like the system only checks efficiency value on the start of an operation.
So if operation start is planned on day shift - its efficiency will be considered 100% even if duration is 24h.
And what I'm trying to get is 100% for the rest of day shift and then 80% for night shift and so on even if the operation is 3 days long.

Is there something I can use to achieve this? May be some different type of resource can be applied?

I also tried to use 2 alternate operations with availability calendars and different operation durations, but it seems to have the same problem - system was choosing and operation at the start and didn't swap to the alternate one on the end of shift.

**Comment by jdetaeye** [comment]
>>... but it seems like the system only checks efficiency value on the start of an operation

Correct, that's the expected behavior.   

The closest alternative is to use an availability calendar to mark some hours in the night shift as unavailable.
